resource-managers/yarn/src/test/scala/org/apache/spark/deploy/yarn/ClientSuite.scala

```diff
@@ -199,6 +206,91 @@ class ClientSuite extends SparkFunSuite with Matchers {
     appContext.getMaxAppAttempts should be (42)
   }
 
+  test("configuration and resource type args propagate " +
```

Comment:
I'd try to come up with shorter names that fit in a single line.

Reply:
fixed